Config/CS.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+	<DynamoServicesPath Condition = "'$(DynamoZeroTouchPath)' == ''">$(SolutionDir)\packages\DynamoVisualProgramming.ZeroTouchLibrary.1.0.0-beta2\lib\net45</DynamoServicesPath>
```

Comment:
Using wrong condition. it should be DynamoServicesPath and not DynamoZeroTouchPath.